Library/brew.sh

```diff
@@ -167,7 +167,7 @@ then
   # Hide shellcheck complaint:
   # shellcheck source=/dev/null
   source "$HOMEBREW_BASH_COMMAND"
-  { "homebrew-$HOMEBREW_COMMAND" "$@"; exit $?; }
+  { "homebrew_$HOMEBREW_COMMAND" "$@"; exit $?; }
```

Comment:
No, I mean escape dash inside HOMEBREW_COMMAND to underline. i.e.

`${HOMEBREW_COMMAND//-/_}`